### PR TITLE
feat: voice messages

### PR DESCRIPTION
**Pull request by jaw0r3k**
**Please describe the changes this PR makes and why it should be merged:**

Backports:
- https://github.com/discordjs/discord.js/pull/9392

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Review by TetieWasTaken — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

File: src/util/Constants.js [hunk 1]
```diff
@@ -1141,6 +1141,12 @@ exports.APIErrors = {
   INVALID_JSON: 50109,
   CANNOT_MIX_SUBSCRIPTION_AND_NON_SUBSCRIPTION_ROLES_FOR_EMOJI: 50144,
   CANNOT_CONVERT_PREMIUM_EMOJI_TO_NORMAL_EMOJI: 50145,
+  VOICE_MESSAGES_DO_NOT_SUPPORT_ADDITIONAL_CONTENT: 50159,
+  VOICE_MESSAGES_MUST_HAVE_A_SINGLE_AUDIO_ATTACHMENT: 50160,
+  VOICE_MESSAGES_MUST_HAVE_SUPPORTING_METADATA: 50161,
+  VOICE_MESSAGES_CANNOT_BE_EDITED: 50162,
+  CANNOT_DELETE_GUILD_SUBSCRIPTION_INTEGRATION: 50163,
```
Comment (kyranet) [inline src/util/Constants.js:1148]: This is not part of the upstream PR nor the docs PR.

**Review by kyranet — APPROVED** [review]

File: src/util/Constants.js (hunk 1 above)
Comment (jaw0r3k) [reply src/util/Constants.js:1148]: It is part of discord api types pr

Comment (Jiralite) [reply src/util/Constants.js:1148]: It's not part of that pull request. It was part of https://github.com/discordjs/discord-api-types/pull/725. You should remove it.

Comment (jaw0r3k) [reply src/util/Constants.js:1148]: Oh I copied too much
Okey, I will delete it

**Pull request merged**